test/bin/scenario.sh

```diff
@@ -17,6 +17,7 @@ source "${SCRIPTDIR}/common.sh"
 DEFAULT_BOOT_BLUEPRINT="rhel-9.2"
 LVM_SYSROOT_SIZE="10240"
 WEB_SERVER_URL="http://${VM_BRIDGE_IP}:${WEB_SERVER_PORT}"
+BOOTC_REGISTRY_URL="${VM_BRIDGE_IP}:5000"
```

Comment:
I wonder if we could use the same registry for both mirrors, but this shouldnt stop the PR.